src/amo/reducers/users.js

```diff
@@ -261,8 +304,29 @@ export const addUserToState = ({ user, state } : {
   return { byID, byUsername };
 };
 
+export const setUserAsEditing = ({ state, userId } : {
```

Comment:
I don't understand the value of these helpers. They include validations via `invariant`, but so do the actions that cause them to be called (e.g., `setUserAsEditing` is called as a result of a `EDIT_USER_ACCOUNT`, and the `editUserAccount` action function which returns the `EDIT_USER_ACCOUNT` action also includes the validation). So the extra validation for `userId` here isn't really necessary.

It seems like it would be simpler to do away with the helper entirely and just inline the code which sets the `isEditingById` state in the `EDIT_USER_ACCOUNT` case of the reducer function.